### PR TITLE
Updates installation instruction for FreeBSD

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Unify the instructions with those for Engine (for FreeBSD) and to make copy-and-past easier,

## Changes

* Adds the same pkg setting as in Zonemaster-Engine.
* Updates to make copy-and-past easier.

## How to test this PR

Perform an installation on FreeBSD.

### CODE REVIEW

**Comment by matsduf** [comment]
@pnax and @mattias-p, please review.

**Review by pnax — APPROVED** [review]

File: docs/Installation.md [hunk 1]
```diff
@@ -127,32 +123,65 @@ sudo systemctl reload apache2
...
+
+#### Update list of package repositories
+Create the file `/usr/local/etc/pkg/repos/FreeBSD.conf` with the following
+content, unless it is already updated:
+
+```
+FreeBSD: {
+url: "pkg+http://pkg.FreeBSD.org/${ABI}/latest",
+}
+```
```
Comment (mattias-p) [inline docs/Installation.md:138]: What would happen if we removed this? (From here and from Engine as well).

Comment (matsduf) [reply docs/Installation.md:138]: Older packages would be installed. I added this when there was an inconsistency between the packages in "monthly", which is the default.

**Comment by matsduf** [comment]
## Testing v2021.2
LGTM.

**Pull request merged**